front/src/Store.ts

```diff
@@ -20,13 +22,25 @@ const reduxDevToolsOptions: Config = {
   },
 };
 
+const buildListenerMiddleware = () => {
```

Comment:
we need to merge that with the config I made in 
https://github.com/osrd-project/osrd/pull/4876/files#diff-e134333a314faf912a3474872afce536f49db9ab4f4dac0f9090cb10dbfa67e3